connector/oidc/oidc.go

```diff
@@ -44,6 +44,12 @@ type Config struct {
 	// InsecureEnableGroups enables groups claims. This is disabled by default until https://github.com/dexidp/dex/issues/1065 is resolved
 	InsecureEnableGroups bool `json:"insecureEnableGroups"`
 
+	// AuthURL provides a way to user overwrite the Auth URL from .well-known/openid-configuration
+	AuthURL string `json:"authURL"`
+
+	// TokenURL provides a way to user overwrite the Token URL from .well-known/openid-configuration
```

Comment:
Could you also add these options to the [documentation](https://github.com/dexidp/website/blob/main/content/docs/connectors/oidc.md)?